src/components/EventHandler.tsx

```diff
@@ -0,0 +1,65 @@
+
+import * as React from "react";
+import * as LibraryEntryPoint from "./LibraryEntryPoint";
+import { LibraryItem } from "./LibraryItem";
+
+/**
+ * The Event class stores the callback function together with a name
+ * that identifies it.
+ */
+export class Event {
+    name: string;
+    callback: Function;
+
+    constructor(name: string) {
+        this.name = name;
+        this.callback = null;
+    }
+
+    registerCallback(callback: Function) {
+        this.callback = callback;
+    }
+}
```

Comment:
And since `Event` has multiple callbacks, then it should contain a method like this:

```ts
export class Event {
    raiseEvent(params?: any | any[]) {
        this.callbacks.forEach(...);
    }
}
```